### PR TITLE
Zhaoyu/doc

### PR DESCRIPTION
**Pull request by DongZhaoYu**
the doc for etcd service.

### CODE REVIEW

File: pai-management/doc/etcd.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+## Deployment
+
+Etcd is deployed as a component of Kubernetes when running this command:
+```bash
+python paictl.py cluster k8s-bootup -p ./path/to/cluster/configuration/dir
+```
+
+## Upgrading and rolling back
+
```
Comment (YanjieGao) [inline pai-management/doc/etcd.md]: suggest tell user how to identify etcd is at which node

File: pai-management/doc/etcd.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+## Goal
+Etcd is a service used by Kubernetes as a consistent and highly-available key value store for all its backing cluster data.
+
+## Build
+OpenPAI doesn't directly build etcd service. The service's image is directly pulled from the docker registry.
+
+## Configuration
+
+Configuration file [kubernetes-configuration.yaml](../../cluster-configuration/kubernetes-configuration.yaml) defines etcd as kubernetes storage and specifies the version.
+- storage-backend: etcd3
```
Comment (YanjieGao) [inline pai-management/doc/etcd.md]: '''yaml
'''

format the config

Comment (DongZhaoYu) [reply pai-management/doc/etcd.md]: add the format as yaml. done

File: pai-management/doc/etcd.md (hunk 1 above)
Comment (DongZhaoYu) [reply pai-management/doc/etcd.md]: add more instructions to identify the etcd nodes and verify its status. done.

**Comment by YanjieGao** [comment]
It's better add dependency introduction to tell user which service use etcd (The service dependency).

**Comment by DongZhaoYu** [comment]
sure, add a new section to describe the data store on etcd. done

**Pull request merged**